examples/aws/ec2.md

```diff
@@ -1,21 +1,77 @@
-# AWS EC2
+---
+runme:
+  id: 01HV6WCDJJZYCVX89YE825JGSF
+  version: v3
+---
 
-Be sure to run through the on-time [setup instructions](setup.md).
+# AWS EC2 Cloud Renderers
 
-List EC2 Instances
+AWS EC2 enables you to deploy cloud-native applications with top-tier security, performance, and reliability.
+
+Discover how you can leverage Runme's robust Notebook Cloud Renderers to engage with your EC2 resources in ways you've never imagined before!
+
+💡 **Important!** Be sure to run through the on-time guide [Getting started with Runme Noteboks for AWS](setup.md).
+
+## List EC2 Instances
+
+One of the fundamental tasks in working with EC2 is to list your instances, serving as the foundation for other operations such as managing instance states—starting, stopping, rebooting, or terminating them—or securely connecting via SSH.
+
+Runme seamlessly integrates with your AWS EC2 resource URLs, mirroring your navigation in the AWS Console directly within your Notebook. This eliminates the need to open the console separately; instead, you can access its functionality right within your Notebook file!
+
+Experience it firsthand by running the following URL to see Runme in action!
```

Comment:
I'd say finish the sentence with a colon rather than an exclamation mark would be more appropriate.